modules/ts/misc/run_suite.py

```diff
@@ -138,10 +144,25 @@ def runTest(self, path, logfile, workingDir, args = []):
             if isColorEnabled(args):
                 args.append("--gtest_color=yes")
             cmd = self.wrapInValgrind([exe] + args)
+            env = {}
+            if not self.options.valgrind and self.options.trace:
+                env['OPENCV_TRACE'] = '1'
+                env['OPENCV_TRACE_LOCATION'] = 'OpenCVTrace-{}'.format(self.getLogBaseName(exe))
```

Comment:
It is located in `modules/core/src/trace.cpp` file (GitHub doesn't load its content due "Large diffs are not rendered by default") - you need to click "Load diffs" button.

There is link to [parameter in .cpp file](https://github.com/opencv/opencv/pull/8766/files#diff-a96ff42ecb4890d5bbe89046ad652922R74)